src/client/game.cpp

```diff
@@ -3887,8 +3888,17 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats, f32 dtime,
 		runData.update_draw_list_timer = 0;
 		client->getEnv().getClientMap().updateDrawList();
 		runData.update_draw_list_last_cam_dir = camera_direction;
+		draw_list_updated = true;
+	}
 
-		updateShadows();
+	if (ShadowRenderer *shadow = RenderingEngine::get_shadow_renderer()) {
+		update_draw_list_delta = shadow->getUpdateDelta();
+
+		if (runData.update_shadows_timer > update_draw_list_delta * (0.99f + 0.0002 * (rand() % 100)) && // 0.99-1.01
```

Comment:
what is the point of this? the behaviour explained by the title would be achieved without it